test/scripts/jenkins_build_load_testing.sh

```diff
@@ -15,10 +15,52 @@ if [[ ! "$TASK_QUEUE_PROCESS_ID" ]]; then
   ./test/scripts/jenkins_xpack_build_plugins.sh
 fi
 
+echo " -> Configure Metricbeat monitoring"
+# Configure Metricbeat monitoring for Kibana and ElasticSearch, ingest monitoring data into Kibana Stats cluster
+# Getting the URL
+TOP="$(curl -L http://snapshots.elastic.co/latest/master.json)"
+MB_BUILD=$(echo $TOP | sed 's/.*"version" : "\(.*\)", "build_id.*/\1/')
+echo $MB_BUILD
+MB_BUILD_ID=$(echo $TOP | sed 's/.*"build_id" : "\(.*\)", "manifest_url.*/\1/')
+
+URL=https://snapshots.elastic.co/${MB_BUILD_ID}/downloads/beats/metricbeat/metricbeat-${MB_BUILD}-linux-x86_64.tar.gz
+URL=https://artifacts.elastic.co/downloads/beats/metricbeat/metricbeat-7.11.0-linux-x86_64.tar.gz
+echo $URL
```

Comment:
@marius-dr , is it something we should take care in future? Do we need to upgrade later?
I wonder if we need to open an issue for it in elastic/kibana-load-testing

Reply:
Until we release 8.0.0, this should be compatible 100%. So that's 1 year from now at least.